testsuite/features/secondary/minxen_guests.feature

```diff
@@ -31,7 +31,6 @@ Feature: Be able to manage XEN virtual machines via the GUI
 
 @virthost_xen
   Scenario: Enable the virtualization host formula for Xen
-    Given I am on the Systems overview page of this "xen_server"
```

Comment:
if you aren't already on a page of the system, how would you see the Formulas tab? Shouldn't we enforce being on such a page?

Reply:
During the previous scenario, you navigate to the overview page of that system.
Since that moment, you are not jumping out of this system page, therefore the Formulas tab still visible.

Internally we use the same web driver session during all the Cucumber feature, only when you run a new feature a new "browser" will open. 
Then only case that could impact on a browser state, is a call to "visit" Capybara method.

Reply:
Then go ahead!